code/modules/projectiles/magazines/specialist.dm

```diff
@@ -288,3 +288,16 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	gun_type = /obj/item/weapon/gun/rifle/railgun
 	icon_state_mini = "mag_railgun"
+
+// pepperball
+
+/obj/item/ammo_magazine/pepperball
+	name = "pepperball canister (SAN balls)"
+	desc = "A canister holding a projectile to be used inside a railgun."
```

Comment:
wouldnt this be used inside a pepperball gun not a railgun?

Reply:
```suggestion
	desc = "A canister meant to hold SAN chemical balls for the pepperball gun.."
```